modules/specialized-guides/pages/salt/salt-connectivity.adoc

```diff
@@ -0,0 +1,156 @@
+In case if clients are losing the connection to the {productname} Server (or {productname} Proxy if involved),
+they are unreachable from the {productname} Server web UI or using the CLI.
+To ensure if it's the case, it's better to check if the client is not reachable from the {productname} Server with [command]``salt MINION_ID test.ping``,
+while [command]``venv-salt-call test.ping`` or ([command]``salt-call test.ping`` in case if non-bundle salt is used on the client)
+is working fine on the client side, it is recommended to set `tcp_keepalive` parameters.
+
+The parameters with the values from the example below could be used as a starting point to look for the better combination, which could work better for certain environment.
+It's recommended to put the parameters in the separate drop-in config file like [path]``/etc/venv-salt-minion/minion.d/tuning-keepalives.conf`` or [path]``/etc/salt/minion.d/tuning-keepalives.conf``, depending on the minion type used on the client side.
+
+.Example: Keepalive parameters example values
+
+----
+######      Keepalive settings        ######
+############################################
+# ZeroMQ now includes support for configuring SO_KEEPALIVE if supported by
+# the OS. If connections between the minion and the master pass through
+# a state tracking device such as a firewall or VPN gateway, there is
+# the risk that it could tear down the connection the master and minion
+# without informing either party that their connection has been taken away.
+# Enabling TCP Keepalives prevents this from happening.
+
+# Overall state of TCP Keepalives, enable (1 or True), disable (0 or False)
+# or leave to the OS defaults (-1), on Linux, typically disabled. Default True, enabled.
+tcp_keepalive: True
+
+# How long before the first keepalive should be sent in seconds. Default 300
+# to send the first keepalive after 5 minutes, OS default (-1) is typically 7200 seconds
+# on Linux see /proc/sys/net/ipv4/tcp_keepalive_time.
+tcp_keepalive_idle: 10
+
+# How many lost probes are needed to consider the connection lost. Default -1
+# to use OS defaults, typically 9 on Linux, see /proc/sys/net/ipv4/tcp_keepalive_probes.
+tcp_keepalive_cnt: 3
+
+# How often, in seconds, to send keepalives after the first one. Default -1 to
+# use OS defaults, typically 75 seconds on Linux, see
+# /proc/sys/net/ipv4/tcp_keepalive_intvl.
+tcp_keepalive_intvl: 10
+----
+
+
+
+
+[[proxies-connectivity]]
+== Proxies Connectivity
+
+[command]``salt-broker`` service is used on {productname} Proxies to forward `salt` traffic between the {productname} Server
+and [command]``salt-minion`` services used on the client side.
+It's possible that [command]``salt-broker`` and all the clients behind it could be affected with the same issue as the clients
+directly connected to the {productname} Server.
+The issue could be fixed with the same parameters as recommended for the minions, but specified in [path]``/etc/salt/broker`` on each {productname} Proxy.
```

Comment:
```suggestion
[command]``salt-broker`` service is used on {productname} Proxies to forward `salt` traffic between the {productname} Server and [command]``salt-minion`` services used on the client side.
It is possible that [command]``salt-broker`` and all the clients behind it could be affected with the same issue as the clients directly connected to the {productname} Server.
The issue could be fixed with the same parameters as recommended for the minions, but specified in [path]``/etc/salt/broker`` on each {productname} Proxy.
```

Reply:
```suggestion
[command]``salt-broker`` service is used on {productname} Proxies to forward `salt` traffic between the {productname} Server and [command]``salt-minion`` service used on the client side.
It is possible that [command]``salt-broker`` and all the clients behind it could be affected with the same issue as the clients directly connected to the {productname} Server.
The issue could be fixed with the same parameters as recommended for the minions, but specified in [path]``/etc/salt/broker`` on each {productname} Proxy.
```

Reply:
Just 1 small change from your suggestion, just noticed not really good typo